src/tests/ftest/container/create.py

```diff
@@ -28,49 +28,35 @@ def test_container_create(self):
 
         :avocado: tags=all,container,tiny,smoke,full_regression,containercreate
         """
-        contuuid = None
-        expected_results = []
-
         # setup the pool
         self.prepare_pool()
-
-        # maybe use the good handle, maybe not
-        handleparam = self.params.get("handle", '/run/poolhandle/*')
-        if handleparam == 'VALID':
-            poh = self.pool.pool.handle
-        else:
-            poh = handleparam
-            expected_results.append('FAIL')
-
-        # maybe use a good UUID, maybe not
-        uuidparam = self.params.get("uuid", "/uuids/*")
-        expected_results.append(uuidparam[1])
-        if uuidparam[0] == 'NULLPTR':
-            contuuid = 'NULLPTR'
-        else:
-            contuuid = uuid.UUID(uuidparam[0])
-
-        should_fail = False
-        for result in expected_results:
-            if result == 'FAIL':
-                should_fail = True
-                break
-
-        try:
-            self.container = DaosContainer(self.context)
-            self.container.create(poh, contuuid)
-
-            # check UUID is the specified one
-            if (uuidparam[0]).upper() != self.container.get_uuid_str().upper():
-                print("uuidparam[0] is {}, uuid_str is {}".format(
-                    uuidparam[0], self.container.get_uuid_str()))
-                self.fail("Container UUID differs from specified at create\n")
-
-            if should_fail:
-                self.fail("Test was expected to fail but it passed.\n")
-
+        self.container = DaosContainer(self.context)
+
+        # Invalid handle => FAIL
+	self.log.info("*** create cont with invalid pool handle")
```

Comment:
(pylint-syntax-error) inconsistent use of tabs and spaces in indentation (<unknown>, line 36)